packages/client/src/runtime/core/model/aggregates/aggregate.ts

```diff
@@ -65,7 +66,7 @@ export function createUnpacker(userArgs: object) {
  * @param modelAction a callback action that triggers request execution
  * @returns
  */
-export function aggregate(client: Client, userArgs: object | undefined, modelAction: ModelAction) {
+export function aggregate(client: Client, userArgs: UserArgs | undefined, modelAction: ModelAction) {
```

Comment:
```suggestion
export function aggregate(client: Client, userArgs: UserArgs, modelAction: ModelAction) {
```

UserArgs type can be undefined I see, so undefined again here is no necessary I think

Reply:
This change might be needed in other places

Reply:
`UserArgs` is not undefined but its properties can.

Reply:
Ok I'm definitely not at my best these 2 days 😅